<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpaceGUI</class>
 <widget class="QMainWindow" name="SpaceGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>384</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>‚ò∞</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>–û—Å–Ω–æ–≤–Ω–æ–µ</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>–ì–µ–æ–º–µ—Ç—Ä–∏—è —Å–µ—Ç–∫–∏</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="2">
            <widget class="QSpinBox" name="SRHeight">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="singleStep">
              <number>8</number>
             </property>
             <property name="value">
              <number>48</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>–í—ã—Å–æ—Ç–∞ —è—á–µ–π–∫–∏</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>–°—Ç—Ä–æ–∫</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>–°—Ç–æ–ª–±—Ü–æ–≤</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="5">
            <widget class="QComboBox" name="tiling">
             <item>
              <property name="text">
               <string>–ü—Ä—è–º–æ—É–≥–æ–ª—å–Ω–∞—è (–æ—Ä—Ç–æ–≥–æ–Ω–∞–ª—å–Ω–∞—è)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–®–µ—Å—Ç–∏—É–≥–æ–ª—å–Ω–∞—è (–≥–µ–∫—Å–∞–≥–æ–Ω–∞–ª—å–Ω–∞—è)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–î–∏–∞–≥—Ä–∞–º–∞ –í–æ—Ä–æ–Ω–æ–≥–æ</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>–¢–∏–ø</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>–®–∏—Ä–∏–Ω–∞ —è—á–µ–π–∫–∏</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="SParam1">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2048</number>
             </property>
             <property name="value">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="SRWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2048</number>
             </property>
             <property name="singleStep">
              <number>8</number>
             </property>
             <property name="value">
              <number>48</number>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QSpinBox" name="SParam2">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2048</number>
             </property>
             <property name="value">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>–ù–∞—Å—Ç—Ä–æ–π–∫–∏ –ª–∞–±–∏—Ä–∏–Ω—Ç–∞</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>–ò–Ω—Ç–µ—Ä–≤–∞–ª –≤–∏–∑—É–∞–ª–∏–∑–∞—Ü–∏–∏</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="mazeAlgDelay">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>–ê–ª–≥–æ—Ä–∏—Ç–º –ø–æ—Å—Ç—Ä–æ–µ–Ω–∏—è</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="isSBSF">
             <property name="text">
              <string>–í–∏–∑—É–∞–ª–∏–∑–∏—Ä–æ–≤–∞—Ç—å</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="mazeAlg">
             <property name="maximumSize">
              <size>
               <width>134</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Recursive backtracker</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ê–ª–≥–æ—Ä–∏—Ç–º –≠–ª–ª–µ—Ä–∞</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–†–∞–Ω–¥–æ–º–∏–∑–∏—Ä–æ–≤–∞–Ω–Ω—ã–π –∞–ª–≥–æ—Ä–∏—Ç–º –ö—Ä–∞—Å–∫–∞–ª–∞</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ê–ª–≥–æ—Ä–∏—Ç–º –ü—Ä–∏–º–∞</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–†–µ–∫—É—Ä—Å–∏–≤–Ω–æ–µ –¥–µ–ª–µ–Ω–∏–µ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ê–ª–≥–æ—Ä–∏—Ç–º –û–ª–¥–æ—Å–∞-–ë—Ä–æ–¥–µ—Ä–∞</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ê–ª–≥–æ—Ä–∏—Ç–º –£–∏–ª—Å–æ–Ω–∞</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hunt-and-Kill</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="mazeInfo">
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>üõà</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QCheckBox" name="isManualStep">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>–ü–æ—Å—Ç—Ä–æ–µ–Ω–∏–µ –Ω–∞ –ø—Ä–æ–±–µ–ª</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>–ù–∞—Å—Ç—Ä–æ–π–∫–∏ –Ω–∞—Ö–æ–∂–¥–µ–Ω–∏—è –∫—Ä–∞—Ç—á–∞–π—à–µ–≥–æ –ø—É—Ç–∏</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QCheckBox" name="isPathAlgVisualization">
             <property name="text">
              <string>–í–∏–∑—É–∞–ª–∏–∑–∏—Ä–æ–≤–∞—Ç—å</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>–ò–Ω—Ç–µ—Ä–≤–∞–ª –≤–∏–∑—É–∞–ª–∏–∑–∞—Ü–∏–∏</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="pathAlgDelay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>185</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="pathAlg">
             <item>
              <property name="text">
               <string>–ü–æ–∏—Å–∫ –≤ —à–∏—Ä–∏–Ω—É</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>–ê–ª–≥–æ—Ä–∏—Ç–º –Ω–∞—Ö–æ–∂–¥–µ–Ω–∏—è –ø—É—Ç–∏</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pathInfo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>üõà</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>–û—Ç–æ–±—Ä–∞–∂–µ–Ω–∏–µ –≥—Ä–∞—Ñ–∏—á–µ—Å–∫–∏—Ö —ç–ª–µ–º–µ–Ω—Ç–æ–≤</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>–°–µ—Ç–∫–∞</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>–†–µ–±—Ä–∞ –≥—Ä–∞—Ñ–∞ –ª–∞–±–∏—Ä–∏–Ω—Ç–∞</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>–ö—Ä–∞—Ç—á–∞–π—à–∏–π –ø—É—Ç—å</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>–¢–æ—á–∫–∏</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>–°—Ç–µ–Ω—ã –ª–∞–±–∏—Ä–∏–Ω—Ç–∞</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_7">
             <property name="text">
              <string>–û—Ç–ª–∞–¥–æ—á–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="apply">
          <property name="text">
           <string>–ü—Ä–∏–º–µ–Ω–∏—Ç—å</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>–ö–∞—Å—Ç–æ–º–∏–∑–∞—Ü–∏—è</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
